modules/encrypt/tests/EncryptSodiumTest.php

```diff
@@ -77,6 +77,83 @@ public function test_decrypt_invalid()
 		$this->assertNull($encrypt->decode(base64_encode('asdasd')));
 	}
 
+	/**
+	 * Test with invalid Key Length and expect exception
+	 * @expectedException Kohana_Exception
+	 * @throws Kohana_Exception
+	 */
+	public function test_invalid_key_length() {
```

Comment:
Those 3 functions beneath perform no asserts

Reply:
no if you look closley you can see in the doc comments expectedException... that's the assert they perform. As we test invalid configuration they expect an exception. Tested it with php unit and test passes. If expectedExceptions wouldn't count as assertion phpunit would throw an error: "No assertion performed" which it doesn't . Therefore this is valid (also according to official PHPUnit Documentation)